x/emissions/module/rewards/rewards_test.go

```diff
@@ -744,10 +743,11 @@ func (s *RewardsTestSuite) getRewardsDistribution(
 		})
 		require.NoError(err)
 	}
-
-	err = actorutils.CloseWorkerNonce(&s.emissionsKeeper, s.ctx, topicId, *inferenceBundles[0].Nonce)
-	s.Require().NoError(err)
-
+	// Advance to close the window
+	newBlock := blockHeight + topic.WorkerSubmissionWindow
```

Comment:
how about to update `blockHeight` here not use `newBlock`? Because we need to increase this `blockHeight` at next.

Reply:
hmmm... although we use blockHeight later as a ref for the reputer nonce (+ gt_lag) so I think I'd keep it as is. However they could bear better names.